brave/browser/brave_browser_context.cc

```diff
@@ -117,7 +148,8 @@ BraveBrowserContext::BraveBrowserContext(const std::string& partition,
       partition_(partition),
       ready_(
         new base::WaitableEvent(base::WaitableEvent::ResetPolicy::MANUAL,
-                            base::WaitableEvent::InitialState::NOT_SIGNALED)) {
+                            base::WaitableEvent::InitialState::NOT_SIGNALED)),
+      delegate_(g_browser_process->profile_manager()) {
```

Comment:
I couldn't find the constructor interface with `ProfileManager`

Reply:
?